src/olympia/devhub/tests/test_views.py

```diff
@@ -2764,13 +2764,15 @@ def test_beta_addon_pass_validation(self, mock_sign_file):
         # Give the add-on an approved version so it can be public.
         version_factory(addon=self.addon)
         self.addon.update(status=amo.STATUS_PUBLIC)
+        existing_file = self.version.all_files[0]
+        existing_file.update(status=amo.STATUS_BETA)
 
-        self.version.all_files[0].update(status=amo.STATUS_BETA)
         self.post(beta=True)
-        file_ = self.version.reload().all_files[0]
+        new_file = File.objects.latest('pk')
```

Comment:
can you do `self.version.files.latest('pk')` instead ? Just to ensure we're hitting the right one.